datamade/erikson-edi
# Add community detail views

## Overview

Add a template, route, and an admin interface for community detail views. Here's what Veronica asked for:

> Provide us with [...] 5 webpages, one for each community, that are similar to the neighborhood detail pages and will contain four sections for the following content:
>
> a. Community Contact Information (name, organization, phone, email)
>
> b. Engaging in EDI (approx..250 word count)
>
> c. Naming the Neighborhoods (approx..600-800 word count)
>
> d. Metadata (this will likely be a link to the PDF but will have a brief paragraph to describe its use)

## Demo

An example for Kankakee, with some short dummy data:

<img width="1158" alt="Screen Shot 2019-07-11 at 4 01 19 PM" src="https://user-images.githubusercontent.com/14170650/61085133-2a175d00-a3f5-11e9-8b3f-4286768afc0b.png">

And the admin interface for editing content:

<img width="1158" alt="Screen Shot 2019-07-11 at 4 02 17 PM" src="https://user-images.githubusercontent.com/14170650/61085189-461afe80-a3f5-11e9-973a-071befd46367.png">

## Testing instructions

- Run migrations: `python manage.py migrate`
- Start a development server: `python manage.py runserver`
- Navigate to http://localhost:8000/admin/edi/
- Select `Partner community contacts`, choose `Add a new partner community contact`, and fill it out
- Navigate back to http://localhost:8000/admin/edi/, select `Partner community page contents`, choose `Add partner community page content`, and fill it out with Lorem Ipsum
- Navigate to http://localhost:8000/, choose the community you added contact/content for, and hit the `Read more about <community>` link in the upper-right-hand corner
- View the detail page and confirm it looks good
